kettle/make_json.py

```diff
@@ -188,7 +188,18 @@ def parse_result(child_node):
         time = float(child_node.attrib.get('time') or 0) #time val can be ''
         failure_text = None
         for param in child_node.findall('failure'):
-            failure_text = param.text or param.attrib.get('message', 'No Failure Message Found')
+            msg_attribute = param.attrib.get('message')
+            msg_data = param.text
+            if msg_attribute and msg_data:
+                # Use both. With Ginkgo V2, the attribute contains the failure message
+                # and the data contains the stack backtrace. We could leave out the
+                # stack backtrace, but the function names in it might be useful
+                # to distinguish different unrelated failures that happen to have the
+                # same message.
+                failure_text = msg_attribute + "\n" + msg_data
+            else:
+                # Use whatever we have, with a default if both are unset.
+                failure_text = msg_attribute or msg_data or 'No Failure Message Found'
```

Comment:
This would be my preference over concatenating the two. It's most likely to bring go.k8s.io/triage back to its previous (known) behavior

Reply:
The previous behavior is "failure message plus backtrace" - it was all one strong.

Reply:
So if we want to restore previous behavior, which also would be my preference unless we have a reason to do differently, we need to concatenate.

Reply:
Sure I meant analogue to behavior for results coming from ginkgo v1 tests... we'll pick out the failure message instead of the stack trace.

Reply:
Example from Ginkgo v1:
```
/nvme/gopath/src/github.com/intel/pmem-csi/test/e2e/deploy/deploy.go:1044
Sep 27 12:55:57.096: create lvm-production PMEM-CSI deployment
Unexpected error:
    <*errors.errorString | 0xc00115ff10>: {
        s: "\"test/setup-deployment.sh\": command failed with no output: fork/exec test/setup-deployment.sh: no such file or directory",
    }
    "test/setup-deployment.sh": command failed with no output: fork/exec test/setup-deployment.sh: no such file or directory
occurred
/nvme/gopath/src/github.com/intel/pmem-csi/test/e2e/deploy/deploy.go:1219
```